Makefile

```diff
@@ -12,7 +12,7 @@ SCIENCE_TFIDF_SVM_MODEL:= tfidf-svm.pkl
 SCIENCE_SCIBERT_MODEL := scibert
 SCIENCE_LABEL_BINARIZER := label_binarizer.pkl
 
-PYTHON := python3
+PYTHON := python3.8
```

Comment:
Why? I thought we'd settled on python3 :)

Reply:
We are going back and forth 😂 When I am creating the requirements with generic python3, its fails on my machine.